create-to-do

biararaujo opened the pull request.

biararaujo committed:
create-to-do

biararaujo committed:
add-import-scss

jhrodriguesit commented on src/App.tsx:
```diff
@@ -1,27 +1,14 @@
-      <div>Header</div>
-      <div>Content</div>
-      <div>
-        {items.map((item: string) => (
```
Great work Bia.
I think if we delete this iteration we will stop showing the todo list in the screen. We want to keep showing the list.

biararaujo replied:
Okay, João! I put it again.

jhrodriguesit commented on src/index.tsx:
```diff
@@ -3,16 +3,15 @@ import ReactDOM from "react-dom/client";
 import "./index.scss";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+import CreateToDo from "./components/CreateToDo/CreateToDo";
```
```suggestion

```
This import should be removed as this component is not being used here.

biararaujo replied:
Done!

jhrodriguesit commented on src/index.tsx:
```diff
@@ -3,16 +3,15 @@ import ReactDOM from "react-dom/client";
-    <App />
-  </React.StrictMode>
-);
+root.render(<React.StrictMode>
```
Are you using the `Prettier` vscode extension? It looks like your your code is not indented properly.

biararaujo committed:
changing the functions

biararaujo committed:
changed the app

putted again the app informations and changed the scss.

biararaujo committed:
include null in the array

biararaujo committed:
delete interface appstate

biararaujo committed:
create index.ts

changed some things in scss.

biararaujo committed:
changed function for arrow function

biararaujo committed:
changed key todo

lucaspereiraio committed:
implementing-stop-button-logic

lucaspereiraio committed:
disable-button-working

now we need to correct the logic, because the button is always disabled now

lucaspereiraio committed:
logic-changes

lucaspereiraio committed:
simplifying-the-logic

lucaspereiraio committed:
jefferson-logics[

Jeffersonlpu committed:
stopAdd-typed-any

lucaspereiraio committed:
solving-typescript

Solving typescript restriction

jhrodriguesit commented on src/components/CreateToDo/CreateToDo.tsx:
```diff
@@ -0,0 +1,55 @@
+  stopAdd,
+}: {
+  onAdd: () => void;
+  stopAdd: () => boolean;
```
Good work! But I am not sure if we need to pass this function as a props for this component. Can we try to do it in a simpler way?

lucaspereiraio replied:
Already done.

jhrodriguesit commented on src/components/CreateToDo/CreateToDo.tsx:
```diff
@@ -0,0 +1,55 @@
+
+  return (
+    <div>
+      {/* {hasNull ? (
```
Could you please remove this comment section?

lucaspereiraio replied:
Already done.

jhrodriguesit reviewed (changes requested):
Could we please resolve the conflict and ensure that this changes are working fine with the latest main branch?

lucaspereiraio committed:
cleaning-code

lucaspereiraio committed:
improving-the-code

Cleaning the code, improving the names of the functions, improving the logic and discarding the comments

biararaujo committed:
changed-the-id-name

biararaujo committed:
Merge branch 'main' into create-to-do

biararaujo committed:
changed-the-name-id

changing the name newItem for todoId and deleted getBackGroundColor from createtodo.

jhrodriguesit committed:
add margin to todo wrapper div
